src/SFML/Main/CMakeLists.txt

```diff
@@ -21,4 +21,6 @@ if(SFML_OS_ANDROID)
 
     # glad sources
     target_include_directories(sfml-main SYSTEM PRIVATE "${PROJECT_SOURCE_DIR}/extlibs/headers/glad/include")
+elseif(SFML_OS_IOS)
+    target_link_libraries(sfml-main PUBLIC SFML::Window)
```

Comment:
Don't think we should create this dependency.

Reply:
It already exists - the main module on iOS requires the app delegate class from the window module, so are you proposing we re-organise the modules so that this dependency isn't necessary?